src/abstractdataframe/abstractdataframe.jl

```diff
@@ -503,8 +503,9 @@ where each row represents a variable and each column a summary statistic.
     - `:all` as the only `Symbol` argument to return all statistics.
     - A `function => name` pair where `name` is a `Symbol` or string. This will
       create a column of summary statistics with the provided name.
-- `cols` : a keyword argument allowing to select only a subset of columns from `df`
-  to describe. Can be any column selector ($COLUMNINDEX_STR; $MULTICOLUMNINDEX_STR).
+- `cols` : a keyword argument allowing to select only a subset or transformation
+  of columns from `df` to describe using `select(df, cols, copycols=false)`.
```

Comment:
Maybe not worth giving the implementation details? Saying that selectors supported by `describe` are accepted is probably enough.
```suggestion
  of columns from `df` to describe.
```
If you agree this can also apply to other places.